tests/integration-tests/tests/api/test_api_client.py

```diff
@@ -136,3 +142,62 @@ def _test_delete_cluster(client, cluster, region):
     response = client.list_clusters(region=region)
     cluster_names = [c["cluster_name"] for c in response["items"]]
     assert_that(cluster_names).does_not_contain(cluster_name)
+
+
+@pytest.mark.usefixtures("instance")
+def test_custom_image(request, region, os, pcluster_config_reader, api_client):
```

Comment:
can you also add a separate test for official images and we have it fail so that we can realize if the feature is broken after the initial release?